docs/book/src/clusterctl/intro.md

```diff
@@ -0,0 +1,278 @@
+# clusterctl
+
+The `clusterctl` CLI tool handles the lifecycle of a cluster-API management cluster.
```

Comment:
Reminder, please use "Cluster API" everywhere, unless you're talking about the GitHub repository, in which case it should be "cluster-api". I would try to avoid using CAPI as an abbreviation, unless it's for a namespace name.

Reply:
done